cgmanifest.json

```diff
@@ -189,6 +198,105 @@
             }
          }
       },
+      {
+         "component":{
+            "type":"git",
+            "git":{
+               "commitHash":"103abd9df2e7ad57f45531349471ea5304a7efe5",
+               "repositoryUrl":"https://github.com/dotnet/corefxlab.git"
+            }
+         }
+      },
+      {
+         "component":{
+            "type":"git",
+            "git":{
+               "commitHash":"18a36291e48808fa7ef2d00a764ceb1ec95645a5",
+               "repositoryUrl":"https://github.com/dotnet/standard.git"
+            }
+         }
+      },
+         {
+         "component":{
+            "type":"git",
+            "git":{
+               "commitHash":"217f52fb121ef92491e5d5f71394b07ce4ead1d0",
+               "repositoryUrl":"https://github.com/KjellKod/g3log.git"
+            }
+         }
+      },
+        {
+         "component":{
+            "type":"git",
+            "git":{
+               "commitHash":"50893291621658f355bc5b4d450a8d06a563053d",
+               "repositoryUrl":"https://github.com/madler/zlib.git"
+            }
+         }
+      },
+           {
+         "component":{
+            "type":"git",
+            "git":{
+               "commitHash":"d264a2603493fecda607c1d1cda87fedba77d36b",
+               "repositoryUrl":"https://github.com/Microsoft/CNTK.git"
+            }
+         }
+      },
+      {
+         "component":{
+            "type":"git",
+            "git":{
+               "commitHash":"9ff0cc0f0212cb1e85b83ce5d2907ba9531b6cae",
+               "repositoryUrl":"https://github.com/nlohmann/json.git"
+            }
+         }
+      },
+            {
+         "component":{
```

Comment:
If the python package is already in 
https://github.com/Microsoft/onnxruntime/blob/master/requirements.txt

Don't put it here.

Reply:
This is registered because https://github.com/Microsoft/onnxruntime/blob/11b369a8641d9dc8989739f3c530a8cf330d7f8b/onnxruntime/core/providers/cpu/math/matmul_helper.h references usage of logic from numpy. It needs to be registered to properly generate the TPN.

Reply:
Ok. It's not a big thing.